CMakeLists.txt

```diff
@@ -171,7 +171,6 @@ enable_testing()
 add_subdirectory(hilti)
 add_subdirectory(spicy)
 add_subdirectory(scripts)
```

Comment:
Let's also remove the fuzzing-related flags in `./configure` and their handling: `--enable-fuzzing` and `--with-clang-fuzzer-no-main`, and their CMake counterparts `USE_FUZZING` and `CLANG_FUZZER_NO_MAIN`.